apps/web/components/Regulations/RegulationChangelog.tsx

```diff
@@ -115,19 +115,24 @@ export const useRegulationEffectPrepper = (
             { name },
           )
           const href =
-            item.effect === 'amend'
+            item.effect === 'amend' && item.status === 'published'
               ? linkToRegulation(regulation.name, {
                   d: item.date,
                   diff: true,
                 })
+              : item.effect === 'amend' && item.status === 'pending'
+              ? linkToRegulation(item.name)
               : undefined
 
           const active = isItemActive(item.date)
           const className = cn(s.smallText, active && s.changelogActive)
 
           const Content = (
             <>
-              <strong>{formatDate(item.date)}</strong>
+              <strong>{formatDate(item.date)}</strong>{' '}
+              {item.status === 'pending' && (
+                <Icon icon="warning" type="outline" size="small" />
+              )}
```

Comment:
Is this enough?

Don't we need to add some sort of label to indicate what the warning is about?

Reply:
The plan was to start with this notification icon and then add something more if needed.